release.js

```diff
@@ -135,22 +149,27 @@ const getTestChanges = () => {
         const oldTestKeys = Object.keys(oldTests);
         const newTestKeys = Object.keys(newTests);
 
-        const added = newTestKeys.filter((k) => !oldTestKeys.includes(k));
-        const removed = oldTestKeys.filter((k) => !newTestKeys.includes(k));
-        const changed = [];
+        const added = newTestKeys
+          .filter((k) => !oldTestKeys.includes(k))
+          .filter(simplifyTestChangesList);
```

Comment:
This will be O(n^2) because `list.includes` is called for east list member. It doesn't matter because the lists will be short, but doing it fast isn't really harder. I'd suggest a helper function that takes a list and returns a new list. First create a set from the list, then for each member of the set check if a shorter version of it is in the set and remove the longer ones. Then convert the set to a list and return.